freeipa/src/test/java/com/sequenceiq/freeipa/converter/stack/CreateFreeIpaRequestToStackConverterTest.java

```diff
@@ -120,9 +138,84 @@ void testConvertForInstanceGroupsWhendiskEncryptionSetIdIsPresent() throws Inter
         ReflectionTestUtils.setField(underTest, "defaultGatewayCidr", Set.of());
         Future<String> owner = CompletableFuture.completedFuture("dummyUser");
 
-        underTest.convert(source, ACCOUNT_ID, owner, "crn1", CloudPlatform.AZURE.name());
-        verify(instanceGroupConverter).convert(any(InstanceGroupRequest.class), anyString(), anyString(), anyString(), anyString(), anyString(),
-                diskEncryptionSetIdCaptor.capture());
-        assertEquals(diskEncryptionSetIdCaptor.getValue(), "dummyDiskEncryptionSetId");
```

Comment:
@doktoric Any reason to remove this verification? Just want to understand if it is intentional.